contents/docs/feature-flags/common-questions.mdx

```diff
@@ -1,90 +1,178 @@
+## How can I reduce the latency of feature flag requests on my server?
 
-> **Note:** Do not use local evaluation in an edge / lambda environment, because this initializes a PostHog instance on every call, which can raise your bill drastically. It's best to use regular flag evaluation here.
+Evaluating feature flags requires making a request to PostHog. However, you can [evaluating feature flags locally](/docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation) instead – without having to make a request to PostHog.
 
----
+## How can I reduce the cost of feature flags?
 
-## On my website, why does the feature flag sometimes flicker?
+If your feature flag usage or costs are too high, there a few methods to reduce them:
 
-By default, flags are loaded from our servers which takes about 100-500ms. During this time, the flag is disabled, which can be the reason why you see things look differently for the first 500ms.
+### 1. Reduce client-side feature flag requests
 
-To fix this, you can [bootstrap feature flags](/docs/feature-flags/bootstrapping-and-local-evaluation).
+In our [JavaScript web](/docs/libraries/js) and [React Native](/docs/libraries/react-native) SDKs, you can reduce the cost of feature flags by reducing the number of requests your client makes to fetch feature flag values for your users.
 
----
+1. Using the the [advanced configuration](/docs/libraries/js#advanced-configuration), set `advanced_disable_feature_flags_on_first_load` to `true`. This stops automatically requesting feature flags whenever a page loads. This is useful when you're immediately calling `posthog.identify()` anyway, which requests flags again.
 
-## I care about latency a lot, and 500ms delays are unacceptable on my servers. Can I do something about this?
+2. Using the the [advanced configuration](/docs/libraries/js#advanced-configuration), set `advanced_disable_feature_flags` to `true`. This stops PostHog automatically requesting feature flags. Instead, use [bootstrapping](/docs/feature-flags/bootstrapping) to load flags exactly once.
 
-Yes, use [local evaluation](/docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation). This downloads flag definitions on your servers and evaluates them locally.
+### 2. Reduce local evaluation costs
 
----
+If you're using [local evaluation](/docs/feature-flags/local-evaluation), your bill may be high because of too many requests to fetch feature flag definitions. By default, PostHog fetches these every 30 seconds.
 
-## My feature flags are sending a lot of events, how can I manage this?
+Each request charges 10 credits, so assuming your server is constantly running and making 2 requests per minute, this will charge `10 * 2 * 60 * 24 * 30 = 864,000 credits` each month.
```

Comment:
Wow, this is a lot 🤔 intimidatingly a lot. How much is free allocation?

Reply:
a million for new users